api-internal/openapi.yaml

```
@@ -113,6 +142,9 @@ paths:
                     type: string
                     enum:
                       - invalid_params
+        # when empty body and no token
+        401:
```

```suggestion
        400:
```

Лучше составить полный список ошибок в формате: https://github.com/atomix-team/codestyle/blob/master/api-conventions.md